docs/Doxyfile.in

```diff
@@ -0,0 +1,2533 @@
+# Doxyfile 1.8.17
+
+# This file describes the settings to be used by the documentation system
+# doxygen (www.doxygen.org) for a project.
+#
+# All text after a double hash (##) is considered a comment and is placed in
+# front of the TAG it is preceding.
+#
+# All text after a single hash (#) is considered a comment and will be ignored.
+# The format is:
+# TAG = value [value, ...]
+# For lists, items can also be appended using:
+# TAG += value [value, ...]
+# Values that contain spaces should be placed between quotes (\" \").
+
+#---------------------------------------------------------------------------
+# Project related configuration options
+#---------------------------------------------------------------------------
+
+# This tag specifies the encoding used for all characters in the configuration
```

Comment:
Man, all these helpful comments sure do make it easy to understand what this file is doing :wink:

Reply:
> lol, tell Doxygen guys about that file cry Should we just leave the ~10 relevant options?

Sorry I was vague, haha. what I meant was "the way doxygen does it is a great example of how commenting the template config file improves discoverability; we should do the same thing in all the Kconfig and CMake files in this repository".